cloud/README.md

```diff
@@ -0,0 +1,6 @@
+# Cloud xOpera examples
+This folder contains xOpera examples that can be used to deploy services on different popular cloud providers. We try to
+provide simple blueprints that show the creation and usage of different cloud resources with xOpera and Ansible. Since
+these examples are in general more complex, they cannot be deployed directly since they require some prerequisites to be
+satisfied first. This means that you will need your cloud account to deploy the solution to a specific cloud provider
+and also you may need to install some other accompanying tools that are used for the deployment process. 
```

Comment:
Could this be solved with TOSCA? So you have the "platform node" as for example is done in particles?

Reply:
Sure, this be simplified like in particles. I propose doing this in one of the following PRs.